app/models/tag.rb

```diff
@@ -1214,6 +1211,53 @@ def indirect_bookmarks(rec=false)
     (work_bookmarks + ext_work_bookmarks + series_bookmarks)
   end
 
+  #################################
+  ## SEARCH #######################
+  #################################
+
+  def unwrangled_query(tag_type, options = {})
+    TagQuery.new(options.merge(
+      type: tag_type,
+      unwrangleable: false,
```

Comment:
I think this part of the query might be broken. The `TagIndexer` uses `as_json(only: [])`, but doesn't include `unwrangleable` as one of the fields. And the `TagQuery` function `unwrangleable_filter` will only add the term query if `options[:unwrangleable].present?`, but `false.present?` is false.